simulator/infra/ports/module.cpp

```diff
@@ -48,6 +48,16 @@ void Module::enable_logging_impl( const std::unordered_set<std::string>& names)
         c->enable_logging_impl( names);
 }
 
+// enable-prefetch function implementation
+void Module::enable_prefetch(bool key, uint32 fetchahead_size, std::string prefetch_method)
+{
+    prefetch_enabled = key;
+    this->fetchahead_size = fetchahead_size;
+    this->prefetch_method = prefetch_method;
```

Comment:
We do not use `this` once it is not needed (see https://github.com/MIPT-ILab/mipt-mips/wiki/Cpp-guidelines#this).

Also, here parameters and members have the same name — warned by Visual Studio.